frontend/src/main.jsx

```diff
@@ -1,19 +1,25 @@
-import React from "react";
 import ReactDOM from "react-dom/client";
 import { createBrowserRouter, RouterProvider } from "react-router-dom";
 import Home from "./pages/home/Home";
 import Welcome from "./pages/Welcome/Welcome";
 import OtherStars from "./pages/otherstars/OtherStars";
+import DescriptionPlus from "./components/DescriptionPlus/DescriptionPlus";
 
 const router = createBrowserRouter([
   {
     path: "/",
     element: <Welcome />,
   },
+
   {
     path: "/home",
     element: <Home />,
   },
+
+  {
+    path: "/DescriptionPlus/:id",
```

Comment:
you guys should think about another name.
and you should not have capitale letter in paths